examples/scripts/vsft_llava.py

```diff
@@ -172,13 +154,13 @@ def __call__(self, examples):
         trainer = SFTTrainer(
             model=model,
             args=training_args,
+            data_collator=collate_fn,
             train_dataset=train_dataset,
             eval_dataset=eval_dataset,
-            dataset_text_field="text",  # need a dummy field
-            tokenizer=tokenizer,
+            tokenizer=processor.tokenizer,
             peft_config=get_peft_config(model_config),
             callbacks=[RichProgressCallback] if TRL_USE_RICH else None,
-            data_collator=data_collator,
+            dataset_text_field="",  # need a dummy field
```

Comment:
i think all these options can go into the `SFTConfig` now

Reply:
done, thanks!